docs/userguide/canvas.rst

```diff
@@ -1232,9 +1232,22 @@ the external monitoring system.
 
     class MonitoringIdStampingVisitor(StampingVisitor):
         def on_signature(self, sig, **headers) -> dict:
-            return {'monitoring_id': uuid4(), 'stamped_headers': ['monitoring_id']}
+            return {'monitoring_id': uuid4().hex}
 
```

Comment:
probably this related parts need to be rebased, right?

Reply:
Definitely.
I saw you merged the other PR, will rebase this one right now.

Reply:
Done.